views/songs/language.pug

```diff
@@ -6,7 +6,7 @@ block content
         each language in languages
             li(id=`${language.code}`)
               span.display
-                a(href="#" onclick="editLanguage(`${language.code}`)") #{language.label}
+                a(href="#" onclick=`editLanguage('${language.code}')`) #{language.label}
```

Comment:
This one is fine, but see line 15. Any event handlers like onclick need to have the update as well.